### PR TITLE
Deploypackage

### PR DESCRIPTION
**Pull request by erocheleau**
Command to deploy a package from a package.xml folder structure.
- [x] There is an associated issue that is labelled
- [x] Code is up-to-date with the `master` branch

**Fixes issue:** https://github.com/coveo/vsforce/issues/9

### CODE REVIEW

**Review by lbenie — APPROVED** [review]
sooooo good

File: src/commands/deploypackageCommand.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          vscode.window.showErrorMessage("Salesforce deploy request error", {title: "Show output", action: "SHOW_OUTPUT"}).then((m: any) => {
+            if (m.action === "SHOW_OUTPUT") {
+              this.output.show();
+            }
+          });
+        }
+      })
+      .catch((reason: string) => {
+        vscode.window.showErrorMessage(reason, {title: "Show output", action: "SHOW_OUTPUT"}).then((m) => {
+          if (m.action === "SHOW_OUTPUT") {
```
Comment (tedre191) [inline src/commands/deploypackageCommand.ts]: Make an Object with an "Execute" method.

File: src/utils/statusBarUtil.ts [hunk 2]
```diff
@@ -10,7 +10,8 @@ export class StatusBarUtil {
   private static promises: { [text: string]: Thenable<any> } = {}
 
   private static statusBarItem = vscode.window.createStatusBarItem(
-    vscode.StatusBarAlignment.Left
+    vscode.StatusBarAlignment.Left,
+    100
```
Comment (tedre191) [inline src/utils/statusBarUtil.ts]: 100 ? Can you make it lower to make it so it won't have more importance than the Git stuff

**Review by tedre191 — CHANGES_REQUESTED** [review]

File: src/utils/statusBarUtil.ts (hunk 2 above)
Comment (lbenie) [inline src/utils/statusBarUtil.ts]: that ⬆️

**Review by lbenie — APPROVED** [review]
Make VSFORCE gr8 again

**Review by tedre191 — APPROVED** [review]

**Pull request merged**